### ISSUE TITLE
i18n of table & page headings, attribute names, links and buttons

### ISSUE THREAD

**Issue by splendeo** [grimen/dry_scaffold#21]
The following command:
    ./script/generate dscaffold Duck name:string about:text _actions:new,create,index,quack

Will generate the following in app/views/duck/index.html.haml

```
= "Resources" # I believe this was meant to say "Ducks"?
...
%th.name= 'Name'
%th.about= 'About'
%th.actions= 'Actions'
```

This makes i18n generation difficult. I suggest ussing human_name & human_attribute_name for the header & attributes, and t('Actions) for the Action header:
    = Duck.human_name(:count => 2)
    ...
    %th.name= Duck.human_attribute_name(:name)
    %th.about= Duck.human_attribute_name(:about)
    %th.actions= t('Actions')

Similarly, the links & buttons can be made i18n-compliant quite easily. The following lines:
    = form.commit_button 'Create'
    = link_to 'Show', resource_path(resource)
    = will_paginate(@collection)
would become:
    = form.commit_button t('Create')
    = link_to t('Show'), resource_path(resource)
    = will_paginate(@collection, :previous_label => t('Previous'), :next_label => t('Next') )

**Comment by grimen** [comment]
Yes it could be enhanced, but I really got no time/motivation to spend on this project right now. I'll gladly accept any patches.
